src/user-management/user-management.spec.ts

```diff
@@ -606,6 +607,9 @@ describe('UserManagement', () => {
         organizationId: 'organization_01H5JQDV7R7ATEYZDEG0W5PRYS',
         userId: 'user_01H5JQDV7R7ATEYZDEG0W5PRYS',
         status: 'active',
+        role: {
```

Comment:
test is expected to fail until feature flag is turned on over launch week. this will not get merged until then but should be ready to ship